### PR TITLE
[14.0][MIG] delivery_postlogistics_dangerous_goods

### PR DESCRIPTION
**Pull request by mmequignon**
follow up of https://github.com/OCA/delivery-carrier/pull/441

Those prs have been merged:
- https://github.com/OCA/community-data-files/pull/109
- https://github.com/OCA/community-data-files/pull/137

### CODE REVIEW

**Comment by mmequignon** [comment]
No idea why the build is failing.
@chienandalu It seems related to `delivery_cttexpress`.
Looks like `http://iberws.tourlineexpress.com:8686/ClientsAPI.svc?singleWsdl` is not responding.
Could you please have a look and confirm?
Thanks!

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
 partner-contact
+community-data-files
```
Comment (simahawk) [inline oca_dependencies.txt]: you can discard this change

**Review by simahawk — APPROVED** [review]

File: oca_dependencies.txt (hunk 1 above)
Comment (simahawk) [reply oca_dependencies.txt]: I deleted it on 14.0 and rebased

**Review by sebalix — APPROVED** [review]

**Pull request merged**